9c-internal/9c-claim-items-test/values.yaml

```diff
@@ -0,0 +1,200 @@
+# Default values for 9c-internal cluster.
+# This is a YAML-formatted file.
+
+
+clusterName: "9c-internal-v2"
+logLevel: "debug"
+
+global:
+  image:
+    repository: planetariumhq/ninechronicles-headless
+    tag: "git-08e7ae4090e459ae16e72d7fa93abc4d5ea84456"
+
+  appProtocolVersion: "1/54684Ac4ee5B933e72144C4968BEa26056880d71/MEUCIQDFvvGTtUSSRj524xJf0EcxmmU6UaXVDAfp3fsb7P6fkwIgdSicMlCnEUZwsPr+W6VLg+2ReY4+FIRC2vB1Yssz7EU=/ZHU5OnRpbWVzdGFtcHUxMDoyMDIzLTA1LTE5ZQ=="
+  genesisBlockPath: "https://release.nine-chronicles.com/genesis-block-9c-main"
+  trustedAppProtocolVersionSigner: "02529a61b9002ba8f21c858224234af971e962cac9bd7e6b365e71e125c6463478"
+  peerStrings:
+  - "033369e95dbfd970dd9a7b4df31dcf5004d7cfd63289d26cc42bbdd01e25675b6f,9c-internal-tcp.planetarium.dev,31234"
+  iceServers:
+  - "turn://0ed3e48007413e7c2e638f13ddd75ad272c6c507e081bd76a75e4b7adc86c9af:0apejou+ycZFfwtREeXFKdfLj2gCclKzz5ZJ49Cmy6I=@turn-us.planetarium.dev:3478"
+
+  networkType: Internal
+  consensusType: pbft
+
+  resetSnapshot: true
+  rollbackSnapshot: false
+  useExternalSecret: true
+
+snapshot:
+  downloadSnapshot: true
+  slackChannel: "9c-internal-claim-items"
+  image: "planetariumhq/ninechronicles-snapshot:git-708a18d12e951f1c4275c5137eb577123dd81715"
+  partition:
+    enabled: true
+    suspend: true
+  path: internal-claim-items
+
+# if you want to delete PVC with the volume provisioned together, set this value "Delete"
+volumeReclaimPolicy: "Retain"
+
+seed:
+  count: 1
+  useTurnServer: false
+  image:
+    repository: planetariumhq/libplanet-seed
+    pullPolicy: Always # Overrides the image tag whose default is the chart appVersion.
+    tag: "git-22bed8ef41ab09258e6404462ec8e07cd038d089"
+
+  hosts:
+  - "9c-internal-tcp.planetarium.dev"
+
+  nodeSelector:
+    node.kubernetes.io/instance-type: m5d.xlarge
+
+validator:
+  count: 4
+  image:
+    repository: planetariumhq/ninechronicles-headless
+    pullPolicy: Always
+
+  consensusSeedStrings:
+  - 033369e95dbfd970dd9a7b4df31dcf5004d7cfd63289d26cc42bbdd01e25675b6f,tcp-seed-1,31235
+
+  hosts:
+  - "9c-internal-claim-item-validator-5.nine-chronicles.com"
+  - "9c-internal-claim-item-validator-6.nine-chronicles.com"
+  - "9c-internal-claim-item-validator-7.nine-chronicles.com"
+  - "9c-internal-claim-item-validator-8.nine-chronicles.com"
+
+  storage:
+    data: 500Gi
+
+  resources:
+    requests:
+      cpu: 1
+      memory: 12Gi
+
+  nodeSelector:
+    eks.amazonaws.com/nodegroup: 9c-internal-r6g_l_2c
+
+  extraArgs:
+  - --tx-quota-per-signer=1
+
+remoteHeadless:
+  image:
+    repository: planetariumhq/ninechronicles-headless
+    pullPolicy: Always
+
+  hosts:
+  - "9c-internal-rpc-1.nine-chronicles.com"
```

Comment:
same as internal

Reply:
Oops... I'll check one more time. Thanks! 🙏🏻

Reply:
@sky1045 I updated another part too. It seems okay to me now. Could you review this pull request again? 🙏🏻